pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/FunctionsImpl.java

```diff
@@ -121,6 +121,23 @@ public void createFunction(FunctionDetails functionDetails, String fileName) thr
         }
     }
 
+    @Override
+    public void createFunctionWithUrl(FunctionDetails functionDetails, String pkgUrl) throws PulsarAdminException {
```

Comment:
Is this used somewhere? Don't we need to modify the CmdFunctions to use this?

Reply:
this feature will be used by #1904 . yes, I was thinking to merge this new API change and then I can make appropriate changes at its usage .